src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -220,7 +220,7 @@ protected String submit(Class<? extends Flight> flightClass, FlightMap parameter
 
   // submit a new job to stairway, wait for it to finish, then return the result
   // protected method intended to be called only from JobBuilder
-  protected <T> T submitAndWait(
+  public <T> T submitAndWait(
```

Comment:
Did you mean to include this change?

Reply:
This change was required so it could be mocked in `SnapshotServiceTest.testPatchSnapshotAuthDomain()`.

I did try adding the `@VisibleForTesting` annotation but then got a warning because it was being used outside the class already; I think that annotation is for `private` members that need more visibility.

Reply:
Sounds good!